tests/test_regularity.py

```diff
@@ -0,0 +1,33 @@
+from pytest import raises
+from lingpy import Wordlist, Alignments
+from lingrex.copar import CoPaR
+from lingrex.util import add_structure
+from lingrex.regularity import regularity
+
+
+dummy_wl = {
+    0: ["doculect", "concept", "form", "ipa", "alignment", "cogid"],
+    1: ["A", "one", "atawu", "atawu", "a t a w u", 1],
+    2: ["B", "one", "atwu", "atwu", "a t - w u", 1],
+    3: ["C", "one", "tawu", "tawu", "- t a w u", 1],
+    4: ["D", "one", "tefu", "tefu", "- t e f u", 1],
+    5: ["A", "two", "satu", "satu", "s a t u", 2],
+    6: ["B", "two", "setu", "setu", "s e t u", 2],
+    7: ["C", "two", "situ", "situ", "s i t u", 2]
+}
```

Comment:
The "official" way to provide data for tests would be `pytest.fixture`, i.e.
```python
@pytest.fixture
def wl():
    return {...}
```
and then
```python
def test_regularity(wl):
    test_wl = Wordlist(wl)
    ...
```

Reply:
Adds a bit verbosity, but would make it easier to move the fixtures to `conftest.py` if they are reused in other test modules, or parametrize tests.

Reply:
@tarotis, if this is not clear, maybe check how this is done already here in the lingrex library, or let us know, so we provide hints on where you can find examples.